abracadabra/src/g4-mandatory/run_action.hh

```diff
@@ -21,10 +21,13 @@ public:
   void   EndOfRunAction(const G4Run*) override;
 
   void AddEdep (G4double edep);
+  void next_event() {evt_number++; }
+  unsigned int get_evt_number() { return evt_number; }
 
 private:
   G4Accumulable<G4double> edep;
   G4Accumulable<G4double> edep2;
+  unsigned int evt_number;
```

Comment:
Looks like we don't need any of this, because G4 already counts events:

   https://gitlab.cern.ch/geant4/geant4/-/blob/master/source/run/include/G4Run.hh#L84-86